package.json

```diff
@@ -4,6 +4,10 @@
   "description": "An HTTP/1.1 client, written from scratch for Node.js",
   "main": "index.js",
   "types": "index.d.ts",
+  "files": [
+    "index.*",
```

Comment:
Could you make this more exact? Something like `index.(js|d.ts)` or whatever the appropriate glob would be? Similarly, the `lib` should be something like `lib/*.js`. Otherwise this is a great change! Happy to reduce our publish size.

@nodejs/undici do we want to ship docs with our code or just the source?